### ISSUE TITLE
Endpoint "GET /user/memberships/orgs/:org" not implemented

### ISSUE THREAD

**Issue by pconrad** [PyGithub/PyGithub#942]
I want to ask "is the current user C a member of organization O."

I tried using the /orgs/:org/members/:user end point and simply specifying the current user as the parameter for `named_user`.   
– GET: github.Organization.Organization.has_in_members()

Using code such as:

```python
        g = Github(resp['access_token'])
        org = g.get_organization(org_name)
        named_user = g.get_user(github_userid)
       isMember = org.has_in_members(named_user)
```

But that isn't working for me, for reasons I can't figure out.  (I used to work, but doesn't any more).
I had a similar problem in a Java app based on https://github.com/jcabi/jcabi-github ... where I got it working using the alternative endpoint:

 "GET /user/memberships/orgs/:org" from 

https://developer.github.com/v3/orgs/members/#get-your-organization-membership

But that one does not seem to be implemented in PyGithub    It wasn't implemented in https://github.com/jcabi/jcabi-github  either, but I was able to find an example of how to "go around" the OO interface and interact with the underlying endpoints.  I'm wondering if there is some similar code for PyGithub?  Something like the PyGithub equivalent of this? 

```java
JsonResponse jr = github.entry()
				.uri().path("/user/memberships/orgs/" + orgname)
				.back()
				.method(Request.GET)
				.fetch()
				.as(JsonResponse.class);

			String role = jr.json().readObject().getString("role");
```

Or, alternatively, perhaps adding a get_org_role(org_name) method to named_user?

**Comment by sfdye** [comment]
I agree, this endpoint needs to be implemented.

For now you can use `org.has_in_members(current NamedUser)` as a work-around. (or `has_in_public_members`)

**Comment by pconrad** [comment]
Unfortunately I need to be able to check regardless of whether the membership is public or not.   The `org.has_in_members(current NamedUser)` was not working for some reason, but to be fair, the day I tried, in retrospect was also the day of the GitHub general service meltdown.  So I will try that again.

Any pointers to code that starts with PyGithub for authentication, but then switches to raw json through the endpoint?

**Comment by sfdye** [comment]
Just add that implementation I guess? Let me know if you need any help

**Comment by sudhir512kj** [comment]
@pconrad I would like to work on it. Assign me.

**Comment by s-t-e-v-e-n-k** [comment]
Closed via #1237
